### PR TITLE
Rs 2023.03.17

### PR DESCRIPTION
**Pull request by rschlaikjer**
Lookin kinda walleyed

2023-03-17-204536_2160x1286_scrot

### CODE REVIEW

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 1]
```diff
@@ -196,37 +203,39 @@ That act alone is enough to make me forget that I'm even alive.
...
-With that in mind, approaching her like this would only lead to such a natural result.
+I chose to approach despite knowing that fact, and this is the natural result.
 }
 [sha:7e5eceb6ffbecc309830e571c37294354ebcb96b]{
 -- Page 239, Offset 40129.
 -- 大型のタンクローリーすら巻き上げ、分解する大気の渦は、いま、愚かにも主の不興をかった小鼠一匹を呑み込もうと迫って来ている。
-The atmospheric vortex, which is even capable of lifting up and disintegrating a large tanker truck, is now coming to consume a single tiny rat that had offended its master.
+The atmospheric vortex,
+ capable of picking up and shredding even a large tanker truck,
+ is now on its way to swallow up a pathetic rat that was foolish enough to displease its master.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:238]: atmospheric vortex > whirlwind
a pathetic rat > the pathetic rat

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 2]
```diff
@@ -196,37 +203,39 @@ That act alone is enough to make me forget that I'm even alive.
...
 }
 [sha:accc8042564424a8f16a3bf4cc34be25beb8f144]{
 -- Page 240, Offset 40130.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:241]: 40131
Remove dashes

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 3]
```diff
@@ -241,17 +250,17 @@ I can't breathe. I can't move my legs.
 [sha:5d31e40716451df2972dd76198041210e1cad568]{
 -- Page 240, Offset 40132.
 -- ちょうどアルクェイドの手の陰になって、あの竜巻からは守られるだろう。
-She's right on the other side of Arcueid's hand, so she'll be protected from this vortex.
+She's on the other side of Arcueid's hand, in the lee, so she should be safe from this vortex.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:253]: vortex > tornado

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 4]
```diff
@@ -241,17 +250,17 @@ I can't breathe. I can't move my legs.
...
 }
 [sha:e104c3586fc33506390c6f54f68f83ecb1ba5ccf]{
 -- Page 240, Offset 40133.
 -- 俺はこのまま、自らの愚かさの代償を払うだけ。
-Right now, I'm the only one who will pay the price for my own foolishness.
+I'm the only one who will pay the price for my foolishness.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:258]: Not entirely sure if this is the intended reading. The above sort of makes one think that, but the construction of the sentence is more like

>Now all that's left for me is to pay the price for my own foolishness.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 5]
```diff
@@ -241,17 +250,17 @@ I can't breathe. I can't move my legs.
...
 }
 [sha:3ce23e25ac444a55f073c8c210279bb9be6cb71b]{
 -- Page 241, Offset 40134.
 -- ……死を受け入れた事による脱力か。
-...Am I exhausted because I've accepted death?
+I feel exhausted... is it because I've accepted my own death?
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:263]: Hmmmmmmmmmmmmm

I don't like it, but I'm not sure how to change it.
It feels wrong.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 6]
```diff
@@ -241,17 +250,17 @@ I can't breathe. I can't move my legs.
...
 }
 [sha:03e994c8e7718cca2c994a743fe0b5a53cd4ebb4]{
 -- Page 241, Offset 40135. Mods: @n.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:266]: This is a sound of her blowing at him

>Phuwww―――

Or something? Not sure.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 7]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 -- ……白い気流の中、無様に落ちていく。
-...I fall awkwardly into the middle of the white air currents.
+I tumble down head over heels, buffeted by the white air currents.
 }
 [sha:86e3a8df03260999f42da435406dccc35a37ab39]{
 -- Page 242, Offset 40139.
 -- 自分が何処にいるのか、落下に備えて何をするべきかも、考えられない。
-I can't think about where I am, or even about how I should prepare for the fall.
+I don't even wonder where I am,
+ or what I should do to try and soften my landing.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:289]: these kind of 'wonder' about か ARE DRIVING ME UP THE WALL

>I don't know where I am, or even what I should do to soften my landing.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 8]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 }
 [sha:9119f355eaf3f228b45135d2953a8d9c5c8e8b3e]{
 -- Page 243, Offset 40140.
 -- ……当然だ。自覚が無かっただけで、何もかも限界だった。
-...That's natural. I've simply lost all self-awareness, so I've practically reached my limit.
+It's only natural...
+ I hadn't noticed, but I was well past my limits.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:295]: >It's only natural.
I'm well past my limits. I just hadn't noticed until now.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 9]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 }
 [sha:711a80bda6e7df935cff010eb72559a1980ff13e]{
 -- Page 243, Offset 40141.
 -- 昨日まで普通に走るだけで息を切らしていたクセに、100メートルを５秒で走るだの、ビルの上を駆け回るだの、はては巨人に跳び移るだの。
-Despite being someone who, from as early as yesterday, would be out of breath from simply running, I ran 100 meters in just 5 second, jumped around on top of buildings, and on top of that, soared my way over to that giant.
+Despite being someone who as of yesterday would be out of breath after a light jog,
+ I just ran a hundred meters in five seconds,
+ dashed around on top of buildings,
+ and jumped all the way over to that giant.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:303]: Just a few fixes with the flow.
>Up until yesterday, I was someone who would lose their breath after a light job, and now I'm running a hundred meters in five seconds, dashing around atop buildings, and even jumping up onto that giant.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 10]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 }
 [sha:f458aa8665eb2b31b1d1c30520ab04de67a9fbed]{
 -- Page 244, Offset 40142.
 -- 全身の筋肉は幾度となく断裂し、その度につなぎ合わされていた。通常の何百倍もの速さの新陳代謝。本来その回復に使われるだけの“時間”を、俺の脳は消費している。
-Every muscle in my body has been torn on multiple occasions, and they were pieced back together each and every time. My metabolism is 100 times faster than normal. The 'time' that would normally be used for recovery is consumed by my brain instead.
+Every muscle in my body has been torn and stitched back together countless times.
+ My metabolism is running hundreds of times faster than normal.
+ My brain is zeroing out the 'time' that would otherwise be needed to heal those wounds.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:310]: The last sentence is about vampire shit, and just do it a bit more literally. "Yeah but that's nonsense". It's fine. It's Nasu's nonsense.
>...been torn asunder and then stitched...
>...
>The 'time' that would ordinarily be used whenever I performed such healing is instead consumed by my brain.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 11]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 -- Page 245, Offset 40143.
 -- 遠野志貴に用意されたリソースはすべて消費された。
-Every resource available to me has been depleted.
+I've used up every resource I had at my disposal.
 }
 [sha:e5eece45007bccbb0915ab82f65bcfbe5d46b45d]{
 -- Page 245, Offset 40144.
 -- 体力も、精神も、この事態を収める為の<方法|プラン>も、すべて。
-My stamina. My mental fortitude. My plan of putting an end to this situation. Everything.
+My stamina, my mental fortitude, my plans for how to put an end to this... everything.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:320]: >...to this——everything.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 12]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 }
 [sha:37483d504ff166685502324e33253fe682cf3209]{
 -- Page 245, Offset 40145.
 -- もういいだろう、と自分に言い聞かせる声がする。
-'It's fine, right?', I try to persuade to myself.
+There's a voice in my head that tells me I did all I could.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:325]: >...that tells me I have done all I could.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 13]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 -- ……だが、本当に？
-...But is that really the case?
+But... is that really true?
 }
 [sha:3701d5375cd8873ebaf50689719335dccf177eb0]{
 -- Page 246, Offset 40148.
 -- 心のどこかで“まだ早い”と首を振る自分がいる。
-Deep down, there's still a part of me that is shaking his head, telling me that it's too soon to give up.
+Deep down, there's a part of me that's shaking his head,
+ telling me that it's still too soon to give up.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:341]: the part of him is still an it

>...that's shaking its head, telling me...

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 14]
```diff
@@ -266,62 +275,70 @@ Right now, I'm the only one who will pay the price for my own foolishness.
...
 }
 [sha:39426c1518c823d97aba73296a5c8010f1ae3426]{
 -- Page 246, Offset 40149.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:344]: >―――You already have the means of dealing with this.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 15]
```diff
@@ -331,30 +348,30 @@ Deep down, there's still a part of me that is shaking his head, telling me that
 [sha:96bfaadc95c8f64bdf7cf195fbce4f2bf23fab50]{
 -- Page 246, Offset 40150.
 -- ―――光体を倒す、という結論は間違いではない。
-―――It isn't wrong to believe that there's a way to defeat that Luminary.
+―――It isn't wrong to decide that the luminary must be defeated.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:351]: >―――Concluding that you have to defeat the Luminary isn't wrong.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 16]
```diff
@@ -331,30 +348,30 @@ Deep down, there's still a part of me that is shaking his head, telling me that
...
 }
 [sha:b63bbb10c044e115e0cfab6edb8946157abd7eb3]{
 -- Page 246, Offset 40151.
 -- ―――間違っているのは、それを成す為の<手|・><順|・>だ。
-―――What's wrong is the way in which I accomplish that.
+―――It is the %{i}means%{/i} to defeat her which are wrong.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:356]: wrong
>It is the %{i}ordering%{i} of how you've chosen to do it that's wrong.

Could use a bit of shortening though

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 17]
```diff
@@ -331,30 +348,30 @@ Deep down, there's still a part of me that is shaking his head, telling me that
...
 -- Page 247, Offset 40154.
 -- それで消耗しきった神経が回復する訳ではないけれど、
-Although it doesn't restore my exhausted nerves ,
+It isn't enough to restore my burned-out nerves―――
 }
 [sha:c3522acc496c20fc853ecab715258042f38d9075]{
 -- Page 247, Offset 40155.
 -- せめて目を開ける程度の覚醒は、促された。
-at the very least, it woke me up to the point where I can open up my eyes.
+But it's at least enough for me to open my eyes.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:376]: This 覚醒 is a more metaphorical awakening, as is common with this word.

>But it's enough of an awakening to at least open my eyes[ to the truth].

last part implied

File: system_strings/sysmes_text.en [hunk 18]
```diff
@@ -5700,7 +5700,7 @@ Ciel returns from her visit to the Church's local branch.
 }
 [sha:d9e00c824599d07d6cf88ca352372319bd627c7d]{
 -- 失敗した以上、猶予はない。^腕の切断を諦めてでも、^先輩を解放しなくては……！
-失敗した以上、猶予はない。^腕の切断を諦めてでも、^先輩を解放しなくては……！
+My failure doesn't leave me^many options. I may have given^up on slicing off her arm, but^I'm still going to free Senpai!
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5703]: >My failure hasn't left me with a lot of time...

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt [hunk 19]
```diff
@@ -21,47 +21,47 @@ Rather than persist with an attack that doesn't work, right now, there's somethi
...
 -- Page 227, Offset 40090.
 -- 「先輩、情けなくてごめん……！
-"Senpai, I'm sorry for being so pathetic...!
+"Senpai... I'm sorry for being so useless!
 }
 [sha:3148034cd5868fbc5b49031cf67c75bd02dced5c]{
 -- Page 227, Offset 40091.
 -- でも助ける、いま助けるから……！」
-But I'm here, I'll save you right away...!"
+But I'm here now, I'm going to save you!"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:39]: Hmm, not sure that connects as well - 
> Sorry I'm useless -> but I've finally made it here! (ceased being useless) 

works nicely as a logical train, whereas this 
> Sorry I'm useless -> but it's fine [that I'm useless (?)]

doesn't make as much sense to me

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 5 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:263]: You could maybe elide the か a bit?
> Strength drains from my limbs as I accept my imminent death.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:266]: Huff? Puff? Blow this house down?

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 7 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:289]: Not sure that suggestion is correct? I thought he was saying "I don't even have the presence of mind _to wonder_ where I am", not just a simple "I don't know where I am"

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 17 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:376]: Feels a little awkward to use awakening as a noun there. If we want to make it more clear that it's figurative, maybe opt for a different phrase?
> But it's enough to lift the veil from my eyes.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 6 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:270]: Puff works I guess

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 7 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:293]: Oh yeah, I was looking too much at the EN and not the JP.

>The questions of where I am or how to soften my landing don't even enter my mind

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 17 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:382]: Sounds cool, sure

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt (hunk 19 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt:40]: I think it's implied what is fine is well, that he's there, etc. People tend to say that when they're rescuing folks.
Anyway, it's not a bit deal. What is a deal concern the punctuation. That at least should be changed

**Pull request merged**